apps/autoscheduler/FunctionDAG.h

```diff
@@ -544,11 +544,6 @@ struct FunctionDAG {
     vector<Node> nodes;
     vector<Edge> edges;
 
-    // We're going to be querying this DAG a lot while searching for
```

Comment:
Interesting, I guess this comment is no longer true because it was slow to index things by Function.